datashuttle/datashuttle.py

```diff
@@ -655,57 +630,128 @@ def _make_directory_trees(
     def _transfer_sub_ses_data(
```

Comment:
At the end of this function, consider printing that a succesfull transfer has occuered.

Reply:
Nice idea, in the end this is handled by rclone that prints to console